### ISSUE TITLE
Add 'external' field to catalog?

### ISSUE THREAD

**Issue by jbusecke** [leap-stc/data-management#58]
Wondering if we should add a bool field `external` to the catalog to distinguish datasets ingested 'in-house' compared to external ones like in #34?

**Comment by cisaacstern** [comment]
I agree some differentiation might be helpful, but what will be most useful for users to know, I wonder? Perhaps where ingestion happened is less important than access considerations (region, provider, requester-pays, etc.) which could be captured in something like tags?

**Comment by jbusecke** [comment]
Ah interesting. I was more thinking of this as an internal field (for LEAP reporting for instance), but I think your points are good ones to @cisaacstern
